sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -1073,7 +1073,7 @@
           <plugin>
             <groupId>net.jonathangiles.tools</groupId>
             <artifactId>dependencyChecker-maven-plugin</artifactId>
-            <version>1.0.5</version> <!-- {x-version-update;net.jonathangiles.tools:dependencyChecker-maven-plugin;external_dependency} -->
+            <version>1.0.6</version> <!-- {x-version-update;net.jonathangiles.tools:dependencyChecker-maven-plugin;external_dependency} -->
```

Comment:
Does this version include the changes you had done for error validation etc?

Reply:
The new version added better support for logging (which helped make dependency validation more performant) and added support for outputting the report into `/target` instead of the directory where the command was ran.

Reply:
Thank you :)